tests/src/JIT/opt/Tailcall/TailcallVerifyWithPrefix.il

```diff
@@ -8795,15 +8797,17 @@
       IL_003a:  call       void [System.Console]System.Console::WriteLine(string)
       IL_003f:  ldstr      "v.i1: {0} != byte.MinValue || v.i2: {1} != short.M"
```

Comment:
I have removed "{0}" and "{1}" in the above string.

Yes, i have changed the way the values get printed.